storage/rocksdb/ha_rocksdb.cc

```diff
@@ -8301,11 +8350,13 @@ int ha_rocksdb::update_sk(const TABLE *const table_arg, const Rdb_key_def &kd,
                                                          old_key_slice);
   }
 
+  // Write unpack_info
+  value_buffer.append(reinterpret_cast<char *>(m_sk_tails.ptr()),
```

Comment:
see my comment below about trying to push the work here into `pack_record` if possible